core/workspace.ts

```diff
@@ -172,22 +172,11 @@ export class Workspace implements IASTNodeLocation {
    */
   private sortObjects_(a: Block|WorkspaceComment, b: Block|WorkspaceComment):
       number {
-    // AnyDuringMigration because:  Property 'getRelativeToSurfaceXY' does not
-    // exist on type 'Block | WorkspaceComment'.
-    const aXY = (a as AnyDuringMigration).getRelativeToSurfaceXY();
-    // AnyDuringMigration because:  Property 'getRelativeToSurfaceXY' does not
-    // exist on type 'Block | WorkspaceComment'.
-    const bXY = (b as AnyDuringMigration).getRelativeToSurfaceXY();
-    // AnyDuringMigration because:  Property 'offset' does not exist on type
-    // '(a: Block | WorkspaceComment, b: Block | WorkspaceComment) => number'.
-    // AnyDuringMigration because:  Property 'offset' does not exist on type
-    // '(a: Block | WorkspaceComment, b: Block | WorkspaceComment) => number'.
-    return aXY.y +
-        (Workspace.prototype.sortObjects_ as AnyDuringMigration).offset *
-        aXY.x -
-        (bXY.y +
-         (Workspace.prototype.sortObjects_ as AnyDuringMigration).offset *
-             bXY.x);
+    const offset =
+        Math.sin(math.toRadians(Workspace.SCAN_ANGLE)) * (this.RTL ? -1 : 1);
```

Comment:
Yes, this is definitely preferable to having an option property on the function object!

The only thing I wonder is if the `.offset` was ever being set to some other value by developers (sort of a monkey patch, given this function is `private`!)  e.g.: @NeilFraser, does Blockly Games ever set this (and can it stop, please?)

Reply:
Blockly Games does not use `.offset`.  I don't recognize that property.